src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import planet from '../images/planet.png';
+
+const Header = () => (
+
+  <header className="header header-section">
+    <div className="logo-text">
+      <img src={planet} alt="planet logo" className="logo" />
+      <h1>Space Traveler&#8217;s Hub</h1>
+    </div>
+    <nav className="navItems">
+      <NavLink
+        className="nav-item"
+        to="/"
+      >
+        {' '}
+        Rockets
+        {' '}
+      </NavLink>
+      <NavLink
+        className="nav-item"
+        to="/missions"
+      >
+        {' '}
+        Missions
+        {' '}
+      </NavLink>
+      <span className="span-icon"> | </span>
+      <NavLink
+        className="nav-item"
+        to="/myProfile"
+      >
+        {' '}
+        My Profile
+        {' '}
+      </NavLink>
+    </nav>
+
+  </header>
```

Comment:
Let's follow the required design wireframe 👍 

> Your webpage header
![image](https://user-images.githubusercontent.com/66112760/151633466-87f6c402-e1d9-4315-a9b2-75468792304d.png)
 
> Required design header
![image](https://user-images.githubusercontent.com/66112760/151633510-86ff0e01-5c1d-48f9-ae58-27a510cff6e6.png)

Reply:
changed